server/repository/src/db_diesel/temperature_log_row.rs

```diff
@@ -30,6 +32,7 @@ table! {
 joinable!(temperature_log -> sensor (sensor_id));
 joinable!(temperature_log -> store (store_id));
 joinable!(temperature_log -> location (location_id));
+joinable!(temperature_log -> temperature_breach (temperature_breach_id));
```

Comment:
Do we keep these when they are not used? Tending to removing them when not needed...